Lib/test/string_tests.py

```diff
@@ -5,6 +5,8 @@
 import unittest, string, sys, struct
 from test import test_support
 from UserList import UserList
+import sys
```

Comment:
I think `sys` is already imported on line 5?

Reply:
I have addressed this.